### DISCUSSION TITLE
Warnungen beim Installieren von eslint

### DISCUSSION THREAD

**Discussion by EllenSteimers** [VDI-CodING/codeING-main#41] [S1D0]
Hallo Zusammen,

bei der Installation von eslint erscheinen bei mir die folgenden Warnungen. 
![eslint](https://user-images.githubusercontent.com/49428575/111871369-ddd23d80-8989-11eb-919e-f1d1b0bbbe74.PNG)

Weiß jemand wie man die beheben kann?

Danke und viele Grüße
Ellen

**Comment by stan1025** [comment]
Hi @EllenSteimers , dein Screenshot verrät mir folgendes:

(1) Die angezeigten Warnungen sind mehr oder weniger Normal. Es werden immer wieder Warnungen kommen, die dich auf besondere Dinge hinweisen. Ich hatte mal ein Ausbilder der gab mir den Tipp - Warnungen sind dazu da um sie zu ignorieren. Du wirst nahezu kein Package finden, wo du keine Warnung angezeigt bekommst. Zusammenfassend in deinem Fall ignorieren.

(2) Dein Screenshot zeigt mir aber auch, dass die Installation zwar erfolgreich war, jedoch in einem Pfad zum einen Falsch ist und zum anderen wirft er bei mir Fragen auf.  Ich hab den Eindruck, dass du in einem falschen Ordner bist. 
Zum einen ist der Befehl 'npm install -D eslint' im Root-Verzeichnis des Repository auszuführen. 
Dem Pfad im Screenshot zufolge bist du tiefer - '...\CodingDojo\Git' ist dein Root-Verzeichnis.

Der zweite Punkt, der mich eher verwundert ist, nach dem Ausführen von 'git clone <CodING-Repo>' sollte im Pfad eigentlich der Name des Repository 'codeING_main' auftauchen - bei dir steht an dieser Stelle jedoch 'git'.

Prüfe daher nochmal, ob du im richtigen Repository bist und danach den Befehl nochmal Ausführen auf Ebene des Root-Verzeichnis.

Falls dich dieser Text mehr verwirrt, als das er hilft, meld dich nochmal bei uns, dann werden wir nochmal ein kurzes Live Meeting machen und es nochmal einrichten und erklären. Uns ist lieber zu Beginn einmal mehr ein bilaterales Live Meeting zu machen als einmal zu wenig, da alles weitere auf dem korrekten Setup aufbaut.
